classes/Subscription.php

```diff
@@ -176,6 +176,15 @@ private function add_subscription($form_data) {
         ]);
     }
 
+    public function invoices() {
+        $invoices = $this->api->getInvoices([
+            'subscription' => $this->stripe->id,
+            'limit' => 24
```

Comment:
Given this do we want some text somewhere to tell people we only display the last 24 invoices?